packages/node-xmpp/test/tls-test.js

```diff
@@ -80,10 +80,6 @@ describe('TLS', function () {
           cl.connection.socket.authorized,
           'Client should have working tls'
         )
-        assert.ok(
-          client.connection.socket.authorized,
-          'Server should have working tls'
-        )
```

Comment:
I'm not sure to see why the server connection socket authorized flag should be set

@lloydwatkin does removing this makes sense to you?